cypress/integration/users/loans/declare-lost.spec.js

```diff
@@ -43,7 +46,7 @@ describe('ui-users-loans: Loans', () => {
       }) => {
         testUserId = userId;
 
-        cy.addServicePointToUser(Cypress.env('servicePoints')[0].id, userId).then(() => {
+        cy.addServicePointToUser([Cypress.env('servicePoints')[0].id], userId).then(() => {
```

Comment:
try to work through spread operator

Reply:
It doesn't work with spread operator

Reply:
try this
![image](https://user-images.githubusercontent.com/93538670/170992308-137cf225-ef2b-43f0-8a3c-2ea785bbf2cd.png)

Reply:
But I have test where I need to create a several service points

Reply:
agree